pcsx2/CDVD/IsoHasher.cpp

```diff
@@ -39,12 +41,18 @@ bool IsoHasher::Open(std::string iso_path, Error* error)
 {
 	Close();
 
+	// Set atomic flag before first CDVD call
+	s_is_computing_hash = true;
```

Comment:
Consider the case that someone tries to rescan all games for the Game List and tries to verify the ISO file of a hash at the same time. From a brief test I did, that seems to deadlock the UI inside CDVD. The correct way to solve this is to use a mutex. You can use std::mutex::try_lock and std::mutex::unlock instead of the atomics here. Make sure to add these calls to the other places that use CDVD too (VM initialization/shutdown and the game list scanner thread). Make sure to call lock and unlock each exactly once, test error paths.